include/ast.h

```diff
@@ -0,0 +1,68 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ast.h                                              :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: dkaiser <dkaiser@student.42heilbronn.de    +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/06/27 11:48:27 by dkaiser           #+#    #+#             */
+/*   Updated: 2024/06/27 12:16:18 by dkaiser          ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+typedef struct s_sequence {
+    struct s_node **nodes;
+} t_sequence;
+
+enum e_node_type {
+ASSIGN,
+PIPE,
+CMD,
+STRING
```

Comment:
The `STRING` type might not be needed, but for now it's better to have it than not.